src/Serialization/Json/ExceptionNormalizer.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Biig\Melodiia\Serialization\Json;
+
+use Symfony\Component\Debug\Exception\FlattenException;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+
+final class ExceptionNormalizer implements NormalizerInterface
+{
+    /** @var bool */
+    private $debug;
+
+    public function __construct(bool $debug = false)
+    {
+        $this->debug = $debug;
+    }
+
+    public function normalize($object, $format = null, array $context = [])
+    {
+        $data = [
+            'type' => 'https://tools.ietf.org/html/rfc2616#section-10',
```

Comment:
What is the point to link the RFC here ? I don't get it

Reply:
Fixed!